<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="resize_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>resize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gray_scale_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>gray_scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="adaptive_Threshold_button">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>adaptive_Threshold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LR_reverse_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>LR_reverse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="canny_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>canny</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dilation_closing_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>dilation closing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="averaging_blurring_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>averaging_blurring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gaussian_blurring_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>gaussian blurring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="median_blurring_button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>median blurring</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CLAHE_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>CLAHE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sharpen_button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>sharpen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bitwise_xor_button">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>bitwise xor</string>
    </property>
   </widget>
   <widget class="QLabel" name="display">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="console">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>461</y>
      <width>341</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
